examples/full_stack/backend/requirements.txt

```diff
@@ -0,0 +1,3 @@
+flask
+flask_sqlalchemy
+embedchain
```

Comment:
Can you please pin the version of these packages that you are using for the playground? We want to make sure that it works fine if someone tries it.